src/java.base/share/classes/java/lang/reflect/code/bytecode/BytecodeLift.java

```diff
@@ -227,7 +232,7 @@ private List<TypeElement> toBlockParams(List<StackMapFrameInfo.VerificationTypeI
                 case ITEM_FLOAT -> params.add(JavaType.FLOAT);
                 case ITEM_DOUBLE -> params.add(JavaType.DOUBLE);
                 case ITEM_LONG -> params.add(JavaType.LONG);
-                case ITEM_NULL -> params.add(JavaType.J_L_OBJECT);
+                case ITEM_NULL -> params.add(JavaType.wildcard());
```

Comment:
That's a curious change, can you explain you did that?

Reply:
I did this purely for debugging purposes. It is hard to differentiate if a variable has J_L_OBJECT type intentionally or as a result of a bug in the merge process of the lift.
I can revert it if needed.

Reply:
It would be odd for that unbounded wildcard type to appear in the model e.g., for `Object o = null`, can it? I recommend hiding it behind a debug system property

Reply:
I think I can remove it now, nulls should be no more an issue.